From 6360f7272a370eee26118d46d963bb91ea36dfa2 Mon Sep 17 00:00:00 2001
From: Vadim Pasternak <vadimp@nvidia.com>
Date: Tue, 29 Sep 2020 08:39:38 +0300
Subject: [PATCH backport v4.19 8/8] mlxsw: i2c: Add device info log

Add info log to probe() routine to report firmware setting in 'dmesg'.

Signed-off-by: Vadim Pasternak <vadimp@mellanox.com>
---
 drivers/net/ethernet/mellanox/mlxsw/i2c.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/net/ethernet/mellanox/mlxsw/i2c.c b/drivers/net/ethernet/mellanox/mlxsw/i2c.c
index 95f408d0e103..e04d521d9376 100644
--- a/drivers/net/ethernet/mellanox/mlxsw/i2c.c
+++ b/drivers/net/ethernet/mellanox/mlxsw/i2c.c
@@ -646,6 +646,11 @@ static int mlxsw_i2c_probe(struct i2c_client *client,
 		return err;
 	}
 
+	dev_info(&client->dev, "Firmware revision: %d.%d.%d\n",
+		 mlxsw_i2c->bus_info.fw_rev.major,
+		 mlxsw_i2c->bus_info.fw_rev.minor,
+		 mlxsw_i2c->bus_info.fw_rev.subminor);
+
 	return 0;
 
 errout:
-- 
2.11.0

